unlock-protocol.com/blog/forbes-ads-free.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Unlocking an ads-free Forbes
+subTitle: Memberships can have diverse benefits!
+authorName: Julien Genestoux
+publishDate: December 17, 2019
+description: Forbes introduced their very first membership today and it lets anyone with an ethereum wallet unlock an ads free experience
+image: /static/images/blog/forbes-ads-free/forbes.svg
+---
+
+At Unlock, we believe that the attention driven economy will be replaced by a membership driven economy. In today's web, creation is rewarded if it is able to attract the eyes, rather than expand the sight: this brought us information overload, endless slideshows, click-bait and fake news.
+
+Of course, more and more people are using ad-blockers, but these come with significant problems: first they starve creators from their revenue streams, and they actually do not fix the incentives. Creators are still paid based on how much attention they can steal... even if it is from fewer and fewer people.
+
+Today, we're proud that [Forbes](https://www.forbes.com/), a publication which writes a lot about the blockchain revolution, is offering the ability to unlock an ads-free experience on their site, using Unlock!
+
+## How to
+
+Forbes deployed 2 locks, the [first one](https://etherscan.io/token/0xf429bb8044bf4b21c65e3ec459eb3010197c6f82) offers a month long ad-free experience on the Forbes.com, while the [second offers](https://etherscan.io/token/0xb2b879764c649c7769f7c90845b0cb2a86add821) the same but only for a week. To purchase a key, you will need to use a crypto wallet such as Opera, MetaMask or Coinbase Wallet. Then, head to The [crypto section of forbes.com](https://www.forbes.com/crypto-blockchain), and click on the "Unlock and ads free experience" button.
+
+TK Screenshot!
+
+An Unlock powered pop-up will open up, allowing you to send a transaction to purchase key! Immediately, even if the transaction has not been mined yet (thanks to optimistic unlocking), you should see the ads disappear, for the duration of the key you bought, on the whole forbes.com website!
+
+Once the transaction has been mined, users will receive a non fungible token which represents their membership! Like any NFT, they can transfer it on store like OpenSea, for example.
+
+![Forbes NFT](/static/images/blog/forbes-ads-free/forbes-nft.png)
+
+## Dog fooding
```

Comment:
I would consider adjusting this heading to something a long the lines of `Experimentation`. While Forbes is using the product they are not dogfooding as they are customers in this case.

https://en.wikipedia.org/wiki/Eating_your_own_dog_food

Reply:
Well I disagree with that. They are dog fooding in a sense that they are also a "customer" of the technology. And the paragrah actually mentions both forbes and "us" crypto-enthusiasts!

Reply:
Scenario 1: Microsoft employees using pre-release betas of Windows to do their day-to-day work (whether they are developers or not)

Scenario 2: A company other than Microsoft choosing to deploy Windows machines to all their employees.

I think that Scenario 1 is dogfooding, and scenario 2 is not. I also think that the Forbes integration is more like scenario 2 than 1.